src/loaders/batch_loader.py

```
@@ -173,6 +179,18 @@ def _mark_batch_failed(self, session, batch: IngestionBatch, error_msg: str) ->
             session.rollback()
             self.logger.error(f"Failed to update batch status: {e}")
 
+    def _get_service_id(self, session, app_id: UUID) -> Optional[UUID]:
+        """app_metadata에서 현재 유효한 service_id 조회."""
+        row = (
+            session.query(AppMetadata.service_id)
+            .filter(AppMetadata.app_id == app_id, AppMetadata.is_active == True)
```

Ruff (0.15.6)

[error] Avoid equality comparisons to `True`; use `AppMetadata.is_active:` for truth checks

Replace with `AppMetadata.is_active`

(E712)